### ISSUE TITLE
Not working anymore with Instagram

### ISSUE THREAD

**Issue by Markussovic** [inDream/DownAlbum#31]
Hello,

your tool is currently not working anymore with Instagram. Please can you check and solve the problem? There's no other tool as smooth as yours was to get full Instagram content easily.

Thanks very much!

**Comment by Jannie25** [comment]
Yes you need to disable the extension and then onlty you can sign into Instagram on Chrome. Some how IG is getting clever to check them. Maybe rename the extension or do something else.

**Comment by Jannie25** [comment]
Or create a stand alone version of the app
